src/webui/src/components/Overview.tsx

```diff
@@ -35,31 +35,31 @@ class Overview extends React.Component<OverviewProps, OverviewState> {
         };
     }
 
-    clickMaxTop = (event: React.SyntheticEvent<EventTarget>) => {
+    clickMaxTop = (event: React.SyntheticEvent<EventTarget>): void => {
         event.stopPropagation();
         // #999 panel active bgcolor; #b3b3b3 as usual
         const { changeMetricGraphMode } = this.props;
         changeMetricGraphMode('max');
     }
 
-    clickMinTop = (event: React.SyntheticEvent<EventTarget>) => {
+    clickMinTop = (event: React.SyntheticEvent<EventTarget>): void => {
         event.stopPropagation();
         const { changeMetricGraphMode } = this.props;
         changeMetricGraphMode('min');
     }
 
-    changeConcurrency = (val: number) => {
+    changeConcurrency = (val: number): void => {
         this.setState({ trialConcurrency: val });
     }
 
-    render() {
+    render(): any {
         const { trialConcurrency } = this.state;
         const { experimentUpdateBroadcast, metricGraphMode } = this.props;
 
         const searchSpace = this.convertSearchSpace();
 
         const bestTrials = this.findBestTrials();
-        const bestAccuracy = bestTrials.length > 0 ? bestTrials[0].accuracy! : NaN;
+        const bestAccuracy = bestTrials.length > 0 ? bestTrials[0].accuracy! : ''; // eslint-disable-line
```

Comment:
when `// eslint-disable-line` is used, please specify the rule such as  `// eslint-disable-line @typescript-eslint/camelcase`, so that we know which rule is disabled

Reply:
Ok.